pgmpy/models/MarkovModel.py

```diff
@@ -177,10 +177,15 @@ def remove_factors(self, *factors):
         for factor in factors:
             self.factors.remove(factor)
 
-    def get_cardinality(self):
+    def get_cardinality(self,node=None):
         """
-        Returns a dictionary with the given factors as keys and their respective
-        cardinality as values.
+        Returns the cardinality of the node. If node is not specified returns
+        a dictionary with the given factors as keys and their respective cardinality
+        as values.
+
+        Parameters
+        ----------
+        node: Any hashable python object(optional).
```

Comment:
@lohani2280 Add a small description about node. Check the docstrings elsewhere to get an idea.